### PR TITLE
document typescript module resolution

### PR DESCRIPTION
**Pull request by ephys**
This PR adds a note in our getting started guide indicating how to configure TypeScript correctly:

image

### CODE REVIEW

**Comment by ephys** [comment]
Is this good enough for #552 or should I add something somewhere else?

**Comment by WikiRik** [comment]
Possibly in pages like https://6504545ac3c0d91d6d9cb066--sequelize-site.netlify.app/docs/v7/models/defining-models/ where we have an info box about the usage of decorators? By referring in that box to the section in getting started?

**Comment by EinfachHans** [comment]
Setting `resolvePackageJsonExports` to `true` doesn't work for me. I had to set the `moduleResolution` (and `module` with it)

**Comment by ephys** [comment]
@WikiRik does this change cover it?

> Setting `resolvePackageJsonExports` to `true` doesn't work for me. I had to set the `moduleResolution` (and `module` with it)

I don't know if there is some interaction between a few different options that make this option incompatible but I think listing both options so people can see which one works for them is the way to go

File: docs/getting-started.mdx [hunk 1]
```diff
@@ -99,6 +99,18 @@ Head to the [Version Policy page](/releases) to know which versions of TypeScrip
 and make sure that the [`@types/node`](https://www.npmjs.com/package/@types/node) package corresponding to your Node.js version is installed
 in your project.
 
+:::info
+
+Sequelize makes use of the the [`exports`](https://nodejs.org/api/packages.html#exports) `package.json` field.
+You may need to update your `tsconfig.json` configuration to make sure TypeScript's module resolution supports it.
+
+At time of writing (TS 5.2), this can be done in two ways:
```
Comment (WikiRik) [inline docs/getting-started.mdx:107]: We can update this to 5.4, but I don't think the settings have changed

**Review by WikiRik — APPROVED** [review]

File: docs/getting-started.mdx (hunk 1 above)
Comment (ephys) [reply docs/getting-started.mdx]: Done

**Review by WikiRik — APPROVED** [review]

**Comment by QuentinFarizon** [comment]
I don't think there really are 2 options, setting `resolvePackageJsonExports` yields :
```
error TS5098: Option 'resolvePackageJsonExports' can only be used when 'moduleResolution' is set to 'node16', 'nodenext', or 'bundler'.
```

**Pull request merged**